File: sortZeroOneTwo/sortZeroOneTwo.py
def sortZeroOneTwo(arr):
    c0 = c1 = c2 = 0
    for num in arr:
        if num == 0:
            c0 += 1
        elif num == 1:
            c1 += 1
        else:
            c2 += 1
    arr[:c0] = [0] * c0
    arr[c0:c0+c1] = [1] * c1
    arr[c0+c1:] = [2] * c2

    print(arr)

sortZeroOneTwo([0,1,2,2,0,1])

File: sortZeroOneTwo/sortZeroOneTwo.js
//Method One Brute Force

function sortZeroOneTwo(arr) {
  let zero = 0;
  let one = 0;
  let two = 0;

  for (let e of arr) {
    if (e == "0") zero++;
    else if (e == "1") one++;
    else two++;
  }
  arr.length = 0;

  for (let i = 0; i < zero; i++) arr.push(0);
  for (let i = 0; i < one; i++) arr.push(1);
  for (let i = 0; i < two; i++) arr.push(2);

  console.log(arr);
}

sortZeroOneTwo([0, 0, 0, 2, 1, 0, 2, 2, 2]);
